copilot/webapp/manifest.yml

```diff
@@ -59,7 +59,7 @@ environments:
       rolling: recreate # Disables blue-green deployment for speed
     variables:
       RAILS_ENV: staging
-      SENTRY_ENVIRONMENT: preview
```

Comment:
I think we'll need to configure Sentry to use `DEPLOY_ENV` as by default it picks up that env var.